### ISSUE TITLE
RadListView - Vue -  Misleading validation message for props _items_

### ISSUE THREAD

**Issue by vratojr** [NativeScript/nativescript-ui-feedback#1190]
When you use an array as collection of items for a RadListView you continously get this error message:

[Vue warn]: Invalid prop: type check failed for prop "items". Expected Object, got Array

This is due to the fact that the prop is defined as:

```
items: {
            type: Object,
            validator: function (val) { return Array.isArray(val) || val instanceof observable_array_1.ObservableArray; },
            required: true
        },
```

Apprently, [if you define a validator, you don't specify a type](https://vuejs.org/v2/guide/components-props.html#Prop-Validation).
Anyway, if you remove the line ` type: Object` the error disappears and validator is still on.:)

### Which platform(s) does your issue occur on?
Both

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI plugin version: 7.0.2**
- **CLI: 6.0.2**
- **Cross-platform modules: 6.0.1**
- **Runtime(s): 6.0.0**

### Please tell us how to recreate the issue in as much detail as possible.
Create a RadListView and pass and array to the _for_ attribute

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
<RadListView ref="listView"  for="el in thisIsAnArray">

**Comment by tsonevn** [comment]
Hi @vratojr,
I've checked the described case, however, was unable to recreate a similar problem on my side. On that matter, can you send us a demo project while using [ {N} Playground](https://play.nativescript.org/), where the issue can be reproduced?

**Comment by sc85** [comment]
I have the same issue.
For reproduction just check the demo you provide in #1182 (https://play.nativescript.org/?template=play-vue&id=BaAIeA&v=6).
<img width="902" alt="Bildschirmfoto 2019-07-31 um 16 49 41" src="https://user-images.githubusercontent.com/4824890/62222206-420a4d00-b3b3-11e9-9d2f-0203927c1499.png">

Running the preview on my Android shows the same warning.

**Comment by zbranzov** [comment]
A fix is released in nativescript-ui-listview@7.0.5
